app/views/posts/index.html.erb

```diff
@@ -1,2 +1,44 @@
-<h1> Post </h1>
-<p> Lists of all posts </p>
\ No newline at end of file
+<div class='center'>
+    <div class='flex '>
+        <div> <%= image_tag @user.Photo %> </div>
+        <div class='border'>
+            <%= @user.Name %>
+            <div class='bottom_flex'>
+                <%= @user.PostCounter %>
+            </div>
+        </div>
+            </div>
+                    <% @posts.each do |post| %>
+ 
+                       <div class='borBtm'>
+                       <div class='border'>
+                            <div class='flex gap2'>
+                            <p>Post:</p>
+                                <%=link_to post.title, user_post_path(@user.id, post.id)%>
```

Comment:
- [ ] Great job implementing most of the requested changes from the previous reviewer, how ever kindly make sure every post shows the id of the post, for example, the first post should have something like `Post #1`

- From the wireframes:
![Screenshot_20230211_115020](https://user-images.githubusercontent.com/25227022/218254069-48960d8c-61c0-4cee-b2ef-80f8a33f879f.png)

- your view looks like this  👇🏼 
![Screenshot_20230211_114618](https://user-images.githubusercontent.com/25227022/218254027-0a8635d4-c04f-43d1-91da-ad6fc9974490.png)